test/index.js

```diff
@@ -474,4 +475,156 @@ describe('HauteCouture', () => {
         });
     });
 
+    it('allows options to be optional', (done, onCleanup) => {
+
+        onCleanup((next) => {
+
+            reset();
+            return next();
+        });
+
+        const server = new Hapi.Server();
+        server.connection();
+
```

Comment:
lab keeps giving me  "no-shadow - 'server' is already declared in the upper scope."
and I'm not sure why.